JPS_BASE_LIB/src/test/java/uk/ac/cam/cares/jps/base/listener/test/BaseChimneyOntologyModelManagerTest.java

```diff
@@ -0,0 +1,67 @@
+package uk.ac.cam.cares.jps.base.listener.test;
+
+import java.io.IOException;
+import java.lang.reflect.Field;
+import java.util.concurrent.ConcurrentHashMap;
+
+import org.apache.jena.ontology.OntModel;
+import org.apache.jena.ontology.OntModelSpec;
+import org.apache.jena.query.QueryExecution;
+import org.apache.jena.query.QueryExecutionFactory;
+import org.apache.jena.query.ResultSet;
+import org.apache.jena.rdf.model.Model;
+import org.apache.jena.rdf.model.ModelFactory;
+import org.apache.jena.rdf.model.Resource;
+import org.junit.Assert;
+import org.junit.Test;
+import org.mockito.MockedStatic;
+import org.mockito.Mockito;
+
+import uk.ac.cam.cares.jps.base.listener.BaseChimneyOntologyModelManager;
+import uk.ac.cam.cares.jps.base.listener.BaseOntologyModelManager;
+
+public class BaseChimneyOntologyModelManagerTest {
+
+	public static final String SPQ_WASTE = "PREFIX j4:<" + "http://www.theworldavatar.com/ontology/ontocape/" + "chemical_process_system/CPS_function/process.owl#> " +
+            "PREFIX j7:<" + "http://www.theworldavatar.com/ontology/" + "meta_model/topology/topology.owl#> " +
+            "SELECT ?wasteStream \r\n" +
+            "WHERE " +
+            "{ ?chimney j7:hasOutput ?wasteStream ." +
+            "  ?wasteStream a j4:NonReusableWasteProduct ." +
+            "}";
+	
+    @Test
+    public void testGetSpeciesMap() throws Exception{
```

Comment:
Exception is never thrown by this test.

Reply:
Resolved in commit https://github.com/cambridge-cares/TheWorldAvatar/pull/28/commits/f222676f217e28905deecb1f70b6e0242d5ddc22